src/FormElement/BaseFormElement.php

```diff
@@ -48,7 +50,7 @@ public function getInputSpecification()
             }
         }
 
-        if (!empty($filters)) {
+        if (count($filters) !== 0) {
```

Comment:
should avoid `count()` here, as it is an `O(n)` operation. Just `if ($filters) {`